index.html

```diff
@@ -2,334 +2,298 @@
-                        <a href="#Contact" class="web-nav-link">Contact</a>
-                    </li>
-                </ul>
-            </nav>
-        </div>
-    </header>
-    <section class="content">
-        <h1 class="box1">I’m Emmanuel
-            Glad to see you!</h1>
-        <p class="box2">
-            I'm a software developer! I can help you build a product,
-            feature or website Look through some of my work and experience!
-            If you like what you see and have a project you need coded, don't hestiate to contact me.
-        </p>
-        <h3 class="box3">
-            LET'S CONNECT
-        </h3>
-        <ul class="box4">
-            <li class="social-media">
-                <img src="images/twitter.png" alt="twitter">
-            </li>
-            <li class="social-media">
-                <img src="images/Linkedin.png" alt="Linkedln">
-            </li>
-            <li class="social-media">
-                <img src="images/m.png" alt="m">
-            </li>
-            <li class="social-media">
-                <img src="images/github.png" alt="github">
-            </li>
-            <li class="social-media">
-                <img src="images/fingers.png" alt="two fingers">
-            </li>
+  <header>
+    <div class="container">
+      <nav class="nav-item1">My Logo</nav>
+      <div class="nav-item3">
+        <span class="hamburger-span" onclick="openNav()"> &#9776; </span>
+      </div>
+      <nav class="web-nav">
+        <ul class="web-nav-ul">
+          <li class="web-nav-list">
+            <a href="#Portfolio" class="web-nav-link">Portfolio</a>
+          </li>
+          <li class="web-nav-list">
+            <a href="#About" class="web-nav-link">About</a>
+          </li>
+          <li class="web-nav-list">
+            <a href="#Contact" class="web-nav-link">Contact</a>
+          </li>
         </ul>
-    </section>
-    <section class="proj-cad" id="Porfolio">
-        <div class="projects">
-            <div class="project-card">
-                <img src="images/Snapshoot2.png" alt="project1">
-            </div>
-            <div class="projects-del">
-
-
-                <h1 class=" title">
-                    Tonic
-                </h1>
-                <div class="role">
-                    <div class="role-name">Canopy</div>
-                    <div class="role-dot"></div>
-                    <div class="role-job">Back End Dev</div>
-                    <div class="role-dot"></div>
-                    <div class="role-year">2015</div>
-                </div>
+      </nav>
+    </div>
+    <div id="myNav" class="overlay">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+      <div class="overlay-content">
+        <a href="#Portfolio" onclick="closeNav()">Portfolio</a>
+        <a href="#About" onclick="closeNav()">About</a>
+        <a href="#Contact" onclick="closeNav()">Contact</a>
+      </div>
```

Comment:
Here also the same 👇 
- [ ] Nice job 👏  👏  On implementing the mobile menu functionality I can see it is working properly, however one of the requirements says that you should use Javascript best practice. I think your project will be well structured if you put all Javascript functionality(events) in your `main.js file`. Kindly remove this `onclick` event here in your HTML and use `addEventListener in `main.js`
#### an example in your main.js
```js
    document.querySelector('.closebtn').addEventListener('click', closeNav)
```